docs/python/reference/contexts.md

```diff
@@ -9,38 +9,52 @@ All are accessed through the syntax `DBOS.<method>` and can only be used once a
 
 ## Context Methods
 
-### send
+### send / send_async
```

Comment:
These changes broke a bunch of anchors--can you fix them?  Or just alias the links back to what they were.